schema/layers/precomposition-layer.json

```diff
@@ -40,6 +40,11 @@
                     "title": "Start Time",
                     "type": "number",
                     "default": 0
+                },
+                "tm": {
+                    "title": "Time Remap",
+                    "description": "Timeline remap function",
+                    "$ref": "#/$defs/properties/scalar-property"
```

Comment:
Technically `tm` when present it must be animated, not sure if we want to enforce this on the schema or not.

Reply:
Tested with a static `tm` property, lottie-web seems to handle it ok: https://skottie.skia.org/3cae01c28c83646d6603e0ddcc3ad55b?h=500&l=true&w=500